app/dashboard/views/alias_contact_manager.py

```diff
@@ -150,6 +151,65 @@ def get_contact_infos(
     return ret
 
 
+def create_contact(alias: Alias, contact_address: str) -> Optional[Contact]:
+    if not current_user.is_premium():
+        flash(f"Please upgrade to premium to create reverse-alias")
+        return None
+
+    try:
+        contact_name, contact_email = parse_full_address(contact_address)
+    except Exception:
```

Comment:
Nits: we can just do `except:` here

Reply:
Should be `ValueError` instead of `Except`. I've replaced it.